community/modules/scheduler/schedmd-slurm-gcp-v5-controller/variables.tf

```diff
@@ -128,6 +128,11 @@ variable "login_startup_scripts_timeout" {
     EOD
   type        = number
   default     = 300
+
+  validation {
+    condition     = var.login_startup_scripts_timeout == 300
+    error_message = "Changes to login_startup_scripts_timeout (default: 300s) are not respected, this will be fixed in a later release"
```

Comment:
Please change to "this is a known issue that will be..."

Reply:
Done